src/Parsing/Impl/Parser.cs

```diff
@@ -2161,7 +2161,7 @@ struct Name {
                 expr = ParseExpression(allowNamedExpressions: false);
             } else {
                 expr = Error(string.Empty);
-                ReportSyntaxError(_lookahead.Span.Start, _lookahead.Span.Start, "expected ':'");
+                ReportSyntaxError(_lookahead.Span.Start, _lookahead.Span.Start, Resources.ExpectedCharacterErrorMsg.FormatInvariant(":"));//"expected ':'"
```

Comment:
No need to format as `:` is always the same here.

Reply:
Made recommended changes.